VARIABLES.md

```diff
@@ -502,15 +518,15 @@ Default:  "{{confluent_server_enabled}}"
 
 ### schema_registry_user
 
-Only use to customize Linux User Schema Registry Service runs with. User must exist on host.
+Only use to customize Linux User Schema Registry Service runs with. Default user is cp-schema-registry
```

Comment:
Modify this variable to customize the Linux User that the Schema Registry Service runs with. Default user is cp-schema-registry.